docs/reference/migration/migrate_6_5.asciidoc

```diff
@@ -103,3 +104,13 @@ positional fields, have be renamed ("prefix" was spliced out):
 - `xpack.security.audit.logfile.prefix.emit_node_name` to
   `xpack.security.audit.logfile.emit_node_name`
 In 7.0.0 the `prefix` variant will be removed.
+
+[float]
+[[breaking_65_analysis_changes]]
+=== Analysis changes
+
+[float]
+==== Deprecated standard_html_strip analyzer
+
+ Deprecated standard_html_strip analyser in 6.5.
+ This will be not create in 7.0 and be removed in 8.0.
```

Comment:
How about:

The `standard_html_strip` analyzer has been deprecated, and should be replaced with a combination of the `standard` tokenizer and `html_strip` char_filter.  Indexes created using this analyzer will still be readable in elasticsearch 7.0, but it will not be possible to create new indexes using it.